### ISSUE TITLE
Something wrong with keyword arguments?

### ISSUE THREAD

**Issue by villares** [berinhard/pyp5js#59]
Running the code below I get a 10x10 rect... I expected a 30x30 rect.

If I use `coisa(n, 30)` I get the 30x30 rect. Also `coisa(n)` gives me an expected 10x10 default behaviour,

```
from pytop5js import *

def setup():
    createCanvas(500, 500)

event_functions = {}.

def draw():
    n = 100
    coisa(n, padrao=30)

def coisa(x, padrao=10):
	rect(x, 10, padrao, padrao)

start_p5(setup, draw, {})
```

**Comment by berinhard** [comment]
Sorry! I've written down the comment but forgot to push the "Comment" button =S

@villares the issue is with how I'm running the transcrypt compiler. It seems I should use a `-k` flag to enable keyword arguments.

**Comment by berinhard** [comment]
Fixed by caa9cb1cba8f3eeb2d1e3b216ec8dbfb89b6ec98. Now, the following sketch is a valid one:

```python
from pytop5js import *

def setup():
    createCanvas(1000, 1000)

def draw():
    fill(255, 0, 0)
    square(20, 20)
    fill(0, 255, 0)
    square(100, 300, size=300)
    fill(0, 0, 255)
    square(500, 500, 100)

def square(x, y, size=50):
    rect(x, y, size, size)
```

Here's the output:

![Screenshot from 2019-06-10 09-39-44](https://user-images.githubusercontent.com/238223/59195955-bcf97900-8b63-11e9-9bd7-6a3f6b362336.png)
